openl3/core.py

```diff
@@ -181,45 +597,293 @@ def process_file(filepath, output_dir=None, suffix=None, model=None,
     content_type : "music" or "env"
         Type of content used to train embedding. Ignored if `model` is
         a valid Keras model.
-    embedding_size : 6144 or 512
+    embedding_size : 8192 or 512
```

Comment:
We should make it clear in the tutorial that the allowed embeddings sizes for audio and image are different